src/screens/About.js

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { Component } from 'react';
 import styled from 'styled-components';
+import { inject, observer } from 'mobx-react';
 import Screen from '../components/Screen';
 
-export default function About() {
```

Comment:
Couldn't you do this with a FC + hooks? `componetDidMount` === `useEffect(() => {}, []);`

Reply:
I tried this but then ran into the following error: Leading decorators must be attached to a class declaration. I think injecting the store into a FC would require the addition of another hook.